scripts/githooks/pre-commit

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Can this hook run `scripts/format.bash`?
That is what Travis runs to check the formatting, so there will be automatically be consistency between the formatting script and the check.